node/src/main/scala/org/bitcoins/node/Node.scala

```diff
@@ -164,13 +164,12 @@ trait Node extends NodeApi with ChainQueryApi with P2PLogger {
         peerMsgSenders(idx).connect()
         val isInitializedF = for {
           _ <- AsyncUtil.retryUntilSatisfiedF(() => isInitialized(idx),
-                                              maxTries = 1024,
+                                              maxTries = 240,
```

Comment:
This will make the tor tests much more flaky, maybe bump it up but only when tor is enabled